Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -1153,6 +1153,7 @@
         SYS_statfs64
         SYS_thread_selfid
         SYS_ulock_wait
+        SYS_ulock_wait2
```

Comment:
Enough of these are common between the files we might want to consider creating another include for them! :-)

Reply:
Yes, that is a good point! We should look into creating a shared list of sys calls.

Thanks for reviewing!